### PR TITLE
[ubuntu-ppa] differentiate distro series for stable and nightly builds

### PR DESCRIPTION
**Pull request by wsnipex**
allows different ubuntu distro series for nightly and stable builds

### CODE REVIEW

**Comment by wsnipex** [comment]
@Rechi ping

File: vars/buildPlugin.groovy [hunk 1]
```diff
@@ -44,10 +44,16 @@ def call(Map addonParams = [:])
...
+	        'stable': [
+	                'focal',
+	                'bionic',
+	                'xenial'
+	        ],
+	        'nightly': [
+	                'groovy',
+	                'focal',
+	                'bionic'
+	        ]
```
Comment (Rechi) [inline vars/buildPlugin.groovy:56]:
```suggestion
		'stable': [
			'focal',
			'bionic',
			'xenial'
		],
		'nightly': [
			'groovy',
			'focal',
			'bionic'
		]
```

File: vars/buildPlugin.groovy [hunk 2]
```diff
@@ -60,6 +66,10 @@ def call(Map addonParams = [:])
 		'Matrix': 'nightly',
 	]
 
+	def distlist = []
+	UBUNTU_DISTS.keySet().each{ k -> distlist.add(UBUNTU_DISTS[k].join(',')) }
+	def all_dists = distlist.join(',').tokenize(',').unique().join(',')
```
Comment (Rechi) [inline vars/buildPlugin.groovy:71]:
```suggestion
	UBUNTU_DISTS.each{ _, dists -> distlist.addAll(dists)}
	def all_dists = distlist.unique().join(',')
```

**Review by Rechi — CHANGES_REQUESTED** [review]
`distlist` and `all_dists` variables have to be renamed to something containing ubuntu.

**Comment by wsnipex** [comment]
thanks, suggestions applied and vars renamed

**Pull request merged**